tools/hangar/src/error.test.ts

```diff
@@ -5,7 +5,8 @@ import { runWingCommand } from "./utils";
 
 errorWingFiles.forEach((wingFile) => {
   test(wingFile, async ({ expect }) => {
-    const args = ["test", "-t", "sim"];
+    const platforms = ["tf-aws"];
+    const args = ["test"];
```

Comment:
Regression (changed from "sim" to "tf-aws")?

Reply:
son of gun, good catch